splight/client/models/bus.go

```diff
@@ -44,14 +44,12 @@ func (m *Bus) FromSchema(d *schema.ResourceData) error {
 		},
 	}
 
-	nominalVoltage := convertAssetMetadata(d.Get("nominal_voltage").(*schema.Set).List())
-	if nominalVoltage.Type == "" {
+	nominalVoltage := convertAssetMetadata(d.Get("nominal_voltage_kv").(*schema.Set).List())
```

Comment:
We might need to rename it later cause Nacho F is renaming this to "nominal_voltage"